src/appleseed.shared/application/progresstilecallback.h

```diff
@@ -29,19 +29,16 @@
 
 #pragma once
 
+// appleseed.shared headers
```

Comment:
Move after appleseed.renderer headers. The order is: from most specific / highest level (i.e. project-specific files) to most general / lowest level (standard header files, platform header files).